mappings/b3c84cec96600d5ff1e2c8aaf1089392c9515840f8795f0b25ea3792636861726c6573776966676c6173736573.json

```diff
@@ -0,0 +1,54 @@
I1Z2ByeGT1rwQeTE1XDVyvjV79hLBERU649fqNhRemzQfbpvqci+74fz4ox12aQqLNJEVV/EqskBadQv0GXl3nTQv+AlYsuHRmpb26iPs9jrNolDDnrWIFJ1/EAZnR7z54giaighWXErCMKpbBOoP6yq2EPuB2M7B5hUoWQcUmnJDyxVXATwor/7tb/bRUerRENqy2OhRy+OsKEro6DolYJW5bV8DKdKodWyXMI6y/qCW74/fwkHXmGqy/l67ihZfEfkBWOJ/DINZpsR1CTF1uiEqnZnXinAGJh+1gJyRZaK23K5e4II8Ia1onzLxCFVgdVffI3g5mkcv2QGEBYdHXIbYDK0KBynxDbIeWvHJKMxV1lSUp/NfQZse5jXPPzEvOUomH7ORjTpDqfsgBKccp4PPUcUGPsiEnaFAW7FArT0bVYEVY8YQYwyqwirQQFu9Hq0tJhVrMXtiF1d+VMXr3Ph9enKbvUSzz7Baw8iCuGxTGpolXlASd/vRfilZOq8kHyL5sdwf1zJyQemfkM56TxXMgoimsuvqh8coCxjrGdvcKVteVlOUXmphZXJJwDyR5uS4yWPGDLco3YKVfujdSk3usbhZXYHktq66sVWBV/+v1PljJcMLig7CiY4iw7ISZxe3ZOdxDYqAH0pzlG0h5mlaU9fIgYb1JMkqwYWuGygpU98wCAVU4kZ2msUzvMIB4FFSbSoqLSVVUJy5NVaQUOCGs3AvQSfsNKtJGGh4ac1h1YXHJc4VFgtW7ON0FMSbrqCl82GIkcfbBr8xKyhIYfJCWfTIn8zU/qDHizKCF58LP9kb1wmonswosS5JIZlj3kVVgJXGAU2ZU+kUXFpwwIntAOcOKrlKePugqqQkT9nLQP600pOgKVjEnhpmQBarGqh49ybIF1ceZ0mBavYRFjYOewiGbn6kRYRK1WZWJXxnn2gxEq3zoz2UlSs0yVdJJxo6u5TrCQlmZMs8gy8y1QeVYtH0IivJfso51SjzgirJiesuwbmCdie5+Nr3ISs1CtMVqavVq8WVI+ZEkjR7zBlKySVCZ1UVEzyNmWbmDA7ASrHQkv1KHgZuzbIE1JzmZmhWtXsAs0RW5sDfIJoxBBbSDWUlXsCoU1UqI7psK60InivjOUhlQ3QhTLQxGTzHh6t5NWb+AsZ1MCFwhRdk+XgipE/XddMJlFdd0LVKYu82plAVWtw6LF6GVYa+ekGKwOu5k+odKXX/i4F5YicjErBTLsqN/9BNWwJKH2Qfn7Utzsi68EAss/t6qpMroVyeDDU83ozSdbZS7ol0ntwJWTuDUthByvDU8EndpizZYygorkKVV2uSPGlkR5wurFqbww66rVeLVDSPPGfaKtGQqxDJGFycEFk5IMgqr/EX4o8sHBat8FlKGxYsDjuH5ymKu96T6l8py1wzFVFbFyqr87Kxjs5iKwohVFVUQxQln1BMMK6SUlPLC8konaVEt60Ek/AhrAqiVLqtRlVPzxbZbdSoNXiCmaywu+IlYDWLl3L22hQvNN1/YCXE4ddkT2wePXkQ9ZiRsVhKxCFa8Zo+zuwWiaKwiFD1noXSdEZJ6IIkFC6ntinc+XdXKAqrWAN7nRZcVdPSW3jaGExLkK6vwIsHaDyFlVlOXG9TtDaMLPX0eVkY1if8dj1x7sS/xascw944UiwFuWsL3rKs0H3QJgYW/CzbKSqTiizuoMfvhADhLbRk0wIgP2qjNQCcWXbks/5Qp8ZETSHVYUyGVyytdlfeGJRMrGagURqoPBtUGT7E/4IOujg6py+TIaWamVUL7+17B2HU1jq6o9P+kMscWVhmQcIdw2WCRCIcVsI7FM8UKMnid6gRejCIndKcCVt0H/7GUurFKvp+FpefnunIsRRo0NWuA0hJCqipLX6T/zAmtvPPdnhj/DSsMVCll1Z49WWktJF+moO1FPJAzLpiA1XW1e98zewhZyd0VsFwhDazMJUq8Irg3VrreQIvaFzGBlQrpU8AK3WBmtUjJgPfVA0EVUpS6msnZCUcBxBnr0kJXAOIk0y1xS6IiYoUVVlFNp24bMwCWsjvTbBuiVZ4jMFBl0FmoMkZ42R/UCPj4kxt2iyBiAWCygBUVmM6KgKWXSDNxyTOcnEVAjnulupWZJoOxD6aabGEZX93QwIe//uREixqp20HRI/+AUI3nQfb6xflkKcnCzG9e7IOwCqppUOnFXRklSnrloftWSL4M8CtkhaiIVWgrw6m9HWTPeFbJ0hbKrfYEKHwQDqVCfVedcJ7FeA5MGXVleFK2BlTZUWGhAd50qqnNaCp8oq0MWK0VJFgFVRDyNZG6L2YFdToJ7SC0Dyaoe/8rkao1Gp1EDEcrUDVV8YHnxK5SaIMVhMLpf1zwOD7igpEVT/igbu5dUGHUrCg0eKQZmxRWw1BXpc8NixkVwJmWvyNvKLxDy1xgJVh4YVPWm25CBZPHWjAE3hOQrKpOqSLrJo9XloKqCFjoCnQv2iBhIDgtMDxws4RUNf6ulj43RIyuOJsboqtWaz+W0eeWuKcuWmfKKbxjUhbBHfdOkrDkX7euXJM5mhasZAyAe1wPHLeQ45AlulPzk7BCqoesWGbN1AWJ0llElS5Qx9NfnsnYfSRtsLoMimjFBoroKnSQFUPPGJrKE6hWU2ClYeuiskMmD0VW/0iuQhBWzFSps58tq61YIax6K8qlXYheDm4pUrUD7Z5oULmmRUEVdAtHrHWdWSrhwAqeKQ5ifBEsTx4D41E2EDgFFoyMB8uL1Jtj0hWSAlxSBstKwNSKi0n+trhjGCVaRVy3bIgEgdZ3TicnHhhYN3ocERa2cia6iq5e0FVgpeNCaTTCylwQWNFAUcM6YjdzRywLC1TR1cIBK2EUVvIppkraCfO3tmcWXg7srLDDaBOtJtupOGI3aEFIjDCwoSnXGohc7uBEWPy01MDJSnlZSbk3aFYSlki9UhbPKIg+c3TlNNRQa3D/G/S9bwfo9moCAAAAAElFTkSuQmCC",
```

Comment:
Hi @CardanoDEVADA,

you are using an image from https://twitter.com/iohk_charles/status/1297679275190112257?lang=bn as your logo. Please, change the logo or close the PR.

Thanks!